analytics/analytics-ui/src/app/results/table/table.component.html

```diff
@@ -0,0 +1,35 @@
+<!--Column Remove Selector and Button-->
+<div>
+  <mat-form-field>
+    <mat-label>Columns</mat-label>
+    <mat-select [(value)]="selected">
+      <mat-option *ngFor="let column of columnsToDisplay" value={{column}}>
```

Comment:
It seems like there are a lot of button clicks needed to remove multiple columns. Perhaps providing a multi-select dropdown would make for a better user experience - similar in design to the Gaffer UI. It looks like the removeColumn() may need a rework too but I'll put more detail on that in the .ts file.